### PR TITLE
[CELEBORN-749] Update grafana dashboard to remove "RSS"

### PR DESCRIPTION
**Pull request by FMX**
### What changes were proposed in this pull request?
Update Grafana dashboard and its setup demo to remove the old name "RSS"

### Why are the changes needed?
Ditto.

### Does this PR introduce _any_ user-facing change?
NO.

### How was this patch tested?
No test needed.

### CODE REVIEW

File: tests/flink-it/src/test/resources/log4j2-test.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
         </File>
     </Appenders>
     <Loggers>
-        <Root level="INFO">
+        <Root level="DEBUG">
```
Comment (AngersZhuuuu) [inline tests/flink-it/src/test/resources/log4j2-test.xml:32]: Why change this?

**Review by AngersZhuuuu — APPROVED** [review]
LGTM

File: tests/flink-it/src/test/resources/log4j2-test.xml (hunk 1 above)
Comment (FMX) [reply tests/flink-it/src/test/resources/log4j2-test.xml:32]: Not needed, I'll revert it.

**Comment by FMX** [comment]
Hold on, I'll reduce image file size.

**Comment by FMX** [comment]
> Hold on, I'll reduce image file size.

Image file size has been reduced.

**Comment by pan3793** [comment]
Thanks, merged to main/0.3

**Pull request closed without merging**